frontend/src/components/user/OpportunityCompetitionStart.tsx

```diff
@@ -13,15 +12,10 @@ export const OpportunityCompetitionStart = ({
   competitions: [Competition];
   opportunityTitle: string;
 }) => {
-  const { user } = useUser();
   const { checkFeatureFlag } = useFeatureFlags();
   const openCompetitions = selectOpenCompetitions({ competitions });
 
-  if (
-    !openCompetitions.length ||
-    !user?.token ||
-    checkFeatureFlag("applyFormPrototypeOff")
-  ) {
+  if (!openCompetitions.length || checkFeatureFlag("applyFormPrototypeOff")) {
```

Comment:
Anon users can now see the start app button and are directed to login if they are not.